build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val runtimeSettings = Seq(
     "org.webjars" % "jquery" % "2.2.0",
 
     "com.datastax.cassandra" % "cassandra-driver-core" % "3.0.0-rc1" exclude("org.xerial.snappy", "snappy-java"),
-    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
+    "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0", // todo remove
```

Comment:
according the docs play already has built in logging, need to investigate the default way